### PR TITLE
Unit Testing setup

### PR DESCRIPTION
**Pull request by akoushke**
Ci config needs to be updated. so wait for the CI PR to get merged first.

### CODE REVIEW

File: src/index.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('To test JEST, will be removed', () => {
+  test('Yo jest, is that you?', () => {
+    expect(true).toBeTruthy();
```
Comment (lalli-flores) [inline src/index.test.js]: 😂

**Comment by lalli-flores** [comment]
Can you remove the rollup commits so we merge the test configuration?

**Comment by akoushke** [comment]
> Can you remove the rollup commits so we merge the test configuration?

I did already

File: src/index.test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('To test JEST, will be removed', () => {
```
Comment (adamweeks) [inline src/index.test.js]: While I do enjoy the tests, I would prefer this to be an actual test describing what we are testing:

describe "Jest functionality"
test "passes a basic truthy test"

**Pull request merged**